src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.tsx

```diff
@@ -28,6 +27,10 @@ function handleChangeDate(
   close();
 }
 
+const Calendar = React.lazy(
+  () => import(/* webpackChunkName: "Calendar" */ './calendarWrapper')
+);
```

Comment:
Is there anywhere in the application that we're using the datePickerField? I can't seem to find one.

Reply:
we are not using it right now, it has a storybook though

Reply:
We should probably send it to the trash instead :smile: